cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -47,8 +47,8 @@ CDAP currently requires Internet access on the CDAP service nodes (or until the
 
 For a Distributed CDAP cluster, version |release|, you **must** install these Hadoop components:
 
-.. |cdh-version|       replace:: 5.0.0 through 5.7.0
-.. |cdh-version-hbase| replace:: 5.0.0 through 5.7.0 (1.2-cdh5.7.0)
+.. |cdh-version|       replace:: 5.1 through 5.8
+.. |cdh-version-hbase| replace:: 5.1 through 5.8 (1.2-cdh5.8.x)
```

Comment:
I don't get what the numbers in parentheses mean, then. Are they the cdap-hbase-compat version needed? If so, the package name has not changed. It is still `cdap-hbase-compat-1.2-cdh5.7.0` even though the 3.5 version will *also* support CDH 5.8's HBase. Did that clear it up?